pcdswidgets/vacuum/base.py

```diff
@@ -151,12 +191,14 @@ def channelsPrefix(self, prefix):
         Parameters
         ----------
         prefix : str
-            The prefix to be used for the channels.
+           The prefix to be used for the channels.
```

Comment:
Nitpick: possible accidental space deletion here? We probably should revert this.
```suggestion
            The prefix to be used for the channels.
```